### PR TITLE
feat: Initialize App

### PR DESCRIPTION
**Pull request by beingeek**

### CODE REVIEW

**Review by larsmaeurer — APPROVED** [review]

**Review by larsmaeurer — APPROVED** [review]

File: german_accounting/hooks.py
Comment (wojosc) [inline german_accounting/hooks.py:1]: @beingeek we will wan't to not have "phamos" in here but rather imat. We will need to check back with the customer on exact details. For now you can leave as is.

File: setup.py
Comment (wojosc) [inline setup.py:1]: @beingeek here also, we should set the customer information rather than "phamos"

**Review by wojosc — APPROVED** [review]

File: setup.py
Comment (beingeek) [reply setup.py:1]: ALright please share email of customer will fix this in a while

File: german_accounting/hooks.py
Comment (wojosc) [reply german_accounting/hooks.py:1]: https://github.com/imat-open-source/German-Accounting/issues/4

**Pull request merged**